src/scripts/stores/user.js

```diff
@@ -74,6 +74,14 @@ export default flux.store({
 			localStorage.setItem('token', payload.token);
 		},
 
+		[Action.User.UpdateName](payload) {
+			localStorage.setItem('user', JSON.stringify(payload.user));
+		},
+
+		[Action.User.UpdatePassword](payload) {
+			localStorage.setItem('user', JSON.stringify(payload.user));
+		},
```

Comment:
Do we really need two Actions that do the exact same thing? Consider just making a Action.User.Update Action, or parse the user json from localstorage, set the name/password then save it again.